File: source/utils/gitignore.py
from io import open
from os import path


def gitignore():
    gitignore_file = '.gitignore'

    gitignore_file_content = '''
node_modules
.vscode
.env
'''

    with open(gitignore_file, 'w') as f:
        f.write(gitignore_file_content)

File: lib/src/functions/gitignore.js
const writeFile = require('../utils/writeFile')

module.exports = async () => {
  const data = {
    gitignoreContent: `node_modules\n.vscode\n.env\n`,
    gitignoreFile: '.gitignore'
  }

  await writeFile(data.gitignoreFile, data.gitignoreContent)
}